src/extensions/WorkspaceView/WorkspaceView.tsx

```diff
@@ -1,21 +1,35 @@
 import React, { FunctionComponent, useMemo } from 'react';
-import { useHistory, useLocation } from 'react-router';
-import { WorkspaceInfo } from '../AppContainer';
-import { Plugins } from '../extensions/extensionInterface';
-import { Recording } from '../reducers/recordings';
-import { ServerInfo } from '../reducers/serverInfo';
-import { Sorting } from '../reducers/sortings';
+import { ExternalSortingUnitMetric, Plugins, Recording, Sorting, SortingCurationAction } from '../extensionInterface';
 import SortingView from './SortingView';
 import WorkspaceRecordingsView from './WorkspaceRecordingsView';
 import WorkspaceRecordingView from './WorkspaceRecordingView';
 
+interface Location {
```

Comment:
Making interfaces for Location and History so that WorkspaceView does not depend on react-router-dom